acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ProcessPantheonNodeRunner.java

```diff
@@ -257,7 +258,7 @@ public boolean isActive(final String nodeName) {
   private void killPantheonProcess(final String name, final Process process) {
     LOG.info("Killing " + name + " process");
 
-    Awaitility.waitAtMost(30, TimeUnit.SECONDS)
+    Awaitility.waitAtMost(60, TimeUnit.SECONDS)
```

Comment:
Did you mean to keep this duration change?

Reply:
Yeah it seems to be slower to shutdown in CI than previously.  The request open ended headers fix and this increase were needed to get tests passing reliably but I wasn't keen on merging with that many changes after the original approval.

Reply:
For the record, shutdown delay was found and fixed in another PR so this duration change wasn't needed.